packages/tools/src/store/ToolGroupManager/ToolGroup.ts

```diff
@@ -216,13 +216,21 @@ export default class ToolGroup {
     if (renderingEngines?.length === 0) {
       throw new Error('No rendering engines found.');
     }
-    if (renderingEngines.length > 1) {
-      throw new Error(
-        'Multiple rendering engines found. You must specify a renderingEngineId.'
-      );
-    }
 
-    const renderingEngineUIDToUse = renderingEngineId || renderingEngines[0].id;
+    // if there are multiple rendering engines, we need to search for the one that
```

Comment:
Can you encapsulate this in a helper method - that just makes the overall function easier to understand.